lib/backend/k8s/conversion/conversion.go

```diff
@@ -446,8 +446,14 @@ func (c Converter) k8sRuleToCalico(rPeers []networkingv1.NetworkPolicyPeer, rPor
 	// TODO: This currently creates a lot of rules by making every combination of from / ports
 	// into a rule.  We can combine these so that we don't need as many rules!
 	for _, port := range ports {
+		protocol, calicoPorts, err := c.k8sPortToCalicoFields(port)
```

Comment:
This is subtly different from what we do in v2.6 - in v2.6 we throw away the entire rule, whereas here it looks like we're just dropping part of the rule which includes the bad port.

Not sure if there's really a significance, but we should probably behave the same.